concepticondata/concepticon.tsv

```diff
@@ -3897,3 +3897,6 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 3911	COW MILK	Food and drink	The milk of cows.	Person/Thing
 3912	BREAST MILK	The body	The milk of humans.	Person/Thing
 3913	COUGH (ACT OF COUGHING)	Basic actions and technology	The act of coughing or the sound of coughing.	Person/Thing	
+3914	TURQUOISE	Sense perception	Having a light greenish blue as its color.	Property	
+3915	PRIDE	Emotions and values	An emotional state of being proud.	Person/Thing	
```

Comment:
Could also be done in this PR I think (if you don't mind):

Link:

- Hale-1973-1798-1116
- Lynott-2013-400-342
- Hill-2015-999-552
- Miklashevsky-2018-506-79
- Zalizniak-2020-2591-2244